apiv2/platformics/graphql_api/core/deps.py

```diff
@@ -38,7 +38,11 @@ async def get_db_session(
     try:
         yield session
     finally:
+        print("===== ")
+        print("===== CLOSING SESSION =====")
+        print("===== ")
         await session.close()  # type: ignore
+    print("===== CLOSING SESSION =====")
```

Comment:
Do we want to retain all of these print statements?

Reply:
I mangled this PR, sorry.